pulp_2to3_migration/tests/functional/test_file_migration.py

```diff
@@ -0,0 +1,99 @@
+                                       ContentFilesApi,
+                                       RepositoriesFileApi,
+                                       RepositoriesFileVersionsApi)
+from pulpcore.client.pulp_2to3_migration import (ApiClient as MigrationApiClient,
+                                                 MigrationPlansApi)
+from pulp_2to3_migration.pulp2.base import Repository as Pulp2Repository, RepositoryContentUnit
+from pulp_2to3_migration.pulp2.connection import initialize
+
+# Can't import ISO model due to PLUGIN_MIGRATORS needing Django app
+# from pulp_2to3_migration.app.plugin.iso.pulp2_models import ISO
+
+
+def monitor_task(tasks_api, task_href):
+    """Polls the Task API until the task is in a completed state.
+
+    Prints the task details and a success or failure message. Exits on failure.
+
+    Args:
+        tasks_api (pulpcore.client.pulpcore.TasksApi): an instance of a configured TasksApi client
+        task_href(str): The href of the task to monitor
+
+    Returns:
+        list[str]: List of hrefs that identify resource created by the task
+
+    """
+    completed = ['completed', 'failed', 'canceled']
+    task = tasks_api.read(task_href)
+    while task.state not in completed:
+        sleep(2)
+        task = tasks_api.read(task_href)
+    if task.state == 'completed':
+        print("The task was successfful.")
+    else:
+        print("The task did not finish successfully.")
+    return task
+
+
+class TestMigrationPlan(unittest.TestCase):
+    """Test the APIs for creating a Migration Plan."""
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Create all the client instances needed to communicate with Pulp.
+        """
+
+        # Initialize MongoDB connection
+        initialize()
+
+        configuration = Configuration()
+        configuration.username = 'admin'
+        configuration.password = 'password'
+        configuration.host = 'http://{}:24817'.format(socket.gethostname())
+        configuration.safe_chars_for_path_param = '/'
+
+        core_client = CoreApiClient(configuration)
+        file_client = FileApiClient(configuration)
+        migration_client = MigrationApiClient(configuration)
+
+        # Create api clients for all resource types
+        # artifacts = ArtifactsApi(core_client)
+        cls.file_repositories = RepositoriesFileApi(file_client)
+        cls.file_repo_versions = RepositoriesFileVersionsApi(file_client)
+        cls.file_content = ContentFilesApi(file_client)
+        # filedistributions = DistributionsFileApi(core_client)
+        # filepublications = PublicationsFileApi(file_client)
+        # fileremotes = RemotesFileApi(file_client)
+        cls.tasks = TasksApi(core_client)
+        # uploads = UploadsApi(core_client)
+        cls.migration_plans = MigrationPlansApi(migration_client)
+
+    def test_migrate_iso_repositories(self):
+        """Test that a valid Migration Plan can be created."""
```

Comment:
It seems like test doesn't run MP validation but it checks few things for the results of the migraiton. 
So I'm not sure if this docsting is up to date.

Reply:
Yes, it's definitely out of date.